src/doc/language.ts

```diff
@@ -325,8 +330,17 @@ export const doc_language:Documentation = {
 			For example, when encountering the sequence of tokens
 			<code class="code">{(</code> then <code class="code">)</code> is fine, while
 			encountering <code class="code">}</code> first indicates a syntax error.
-			</p>
-		`,
+			<tscript do-not-run>
```

Comment:
Not sure from the context displayed in the preview: should this really be a tscript tag or better an ebnf tag?